node-fs-readfile/cat.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+for (let index = 2; index < process.argv.length; index++) {
+  fs.readFile(process.argv[index], 'utf8', (err, data) => {
+    if (err) {
+      return;
+    }
+    console.log(data);
+  });
+}
```

Comment:
This will read all the files the user inputs, but it will not guarantee the order the files are output in. Part of the requirement for this bonus challenge is that the files always print out in the same order that the user input them.